dataProcessing.py

```diff
@@ -4,7 +4,7 @@
 import sys
 
 # Parse HHSearch output file up to probability cutoff
```

Comment:
These ought really to go as docstrings on the functions rather than comments, but that's a minor point :)

Reply:
#23